images/scripts/install_daos.sh

```diff
@@ -0,0 +1,262 @@
+#!/bin/bash
+#
+# Install DAOS Server or Client packages
+#
+
+set -e
+trap 'echo "An unexpected error occurred. Exiting."' ERR
+
+SCRIPT_NAME=$(basename "$0")
+DAOS_REPO_BASE_URL="${DAOS_REPO_BASE_URL:-https://packages.daos.io}"
+
+show_help() {
+  cat <<EOF
+Usage:
+
+  ${SCRIPT_NAME} <options>
+
+  Install the DAOS server or client
+
+Options:
+
+  -t --type     DAOS_INSTALL_TYPE   Installation Type
+                                    Valid values [ all | client | server ]
+                                    all = Install both client and server
+
+  -v --version  DAOS_VERSION        Version of DAOS to install from
+                                    https://packages.daos.io/
+                                    If https://packages.daos.io/v2.0.0
+                                    then --version "2.0.0"
+
+  [-u --repo-baseurl DAOS_REPO_BASE_URL ]
+                                    Base URL of a repo.
+                                    This is the URL up to the version.
+                                    If the repo is at https://example.com/foo/v2.0.0
+                                    then -u "https://example.com/foo"
+
+  [ -h --help ]                     Show help
+
+Examples:
+
+  Install daos-client 2.0.0
+
+    ${SCRIPT_NAME} -t client -v 2.0.0
+
+    OR
+
+    export DAOS_INSTALL_TYPE="client"
+    export DAOS_VERSION="2.0.0"
+    ${SCRIPT_NAME}
+
+  Install daos-server 2.0.0
+
+    ${SCRIPT_NAME} -t server -v 2.0.0
+
+    OR
+
+    export DAOS_INSTALL_TYPE="server"
+    export DAOS_VERSION="2.0.0"
```

Comment:
This is is very user accessible.  We don't want users doing this. "export DAOS_VERSION="2.0.0""  Can you better hid this dev feature.

Reply:
I'm not understanding the reasoning behind trying to hide how the version number is set. 

Are you asking me to just remove that from the help output or remove the ability to pass the version in an environment variable?

Probably deserves an offline discussion.

Reply:
@markaolson  After some thought lets just leave this as an undocumented feature.   

The core issue is we do not want DAOS GCP USERs playing around with DAOS Versions.  This should be an Experts only type activity.  As you can just see there are DAOS 2.0 Lib fabric  changes getting landed.   If a USE decide DAOS 1.2 or 1.0 would be fun to try bad things would happen.